ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python3 --version
+pushd /tmp
+
+ls -la /root/kraken
+git clone https://github.com/redhat-chaos/krkn-hub.git
+pushd krkn-hub/
+
+echo "kubeconfig loc $$KUBECONFIG"
+
+export KRKN_KUBE_CONFIG=$KUBECONFIG
+
+./prow/container-scenarios/prow_run.sh
```

Comment:
Please add a line after calling `prow_run.sh` to indicate the execution is completed, something like `echo Done. Return: $?`.

Reply:
Done! Rerunning the test case. Thanks.